var/spack/repos/builtin/packages/quantum-espresso/package.py

```diff
@@ -61,23 +60,44 @@ class QuantumEspresso(CMakePackage):
     with when("+cmake"):
         depends_on("cmake@3.14.0:", type="build")
         conflicts("@:6.7", msg="+cmake works since QE v6.8")
-        # TODO
-        # variant(
-        #     'gpu', default='none', description='Builds with GPU support',
-        #     values=('nvidia', 'none'), multi=False
-        # )
 
     variant("libxc", default=False, description="Uses libxc")
     depends_on("libxc@5.1.2:", when="+libxc")
 
-    variant("openmp", default=False, description="Enables openMP support")
+    variant("openmp", default=True, description="Enables OpenMP support")
     # Need OpenMP threaded FFTW and BLAS libraries when configured
     # with OpenMP support
     with when("+openmp"):
-        conflicts("^fftw~openmp")
-        conflicts("^amdfftw~openmp")
-        conflicts("^openblas threads=none")
-        conflicts("^openblas threads=pthreads")
+        depends_on("fftw+openmp", when="^fftw")
+        depends_on("amdfftw+openmp", when="^amdfftw")
+        depends_on("openblas threads=openmp", when="^openblas")
+        depends_on("amdblis threads=openmp", when="^amdblis")
+
+    # Add cuda support
```

Comment:
Let us mention "CUDA Fortran" instead of CUDA. Use upper case.
Also add a note that it depends on NVHPC compiler but not directly on CUDA toolkit.